src/tools/annotation/bidirectionalTool/utils/updatePerpendicularLineHandles.js

```diff
@@ -7,6 +7,9 @@ export default function(eventData, data) {
   let startX, startY, endX, endY;
 
   const { start, end } = data.handles;
+  const { columnPixelSpacing, rowPixelSpacing } = eventData.image;
```

Comment:
I think in this instance, that this is fine because the `cps` and `rps` are used near this, and the meaning is pretty clear. In general, I like keeping the longer, descriptive variable names.

`const { columnPixelSpacing = 1, rowPixelSpacing = 1 } = eventData.image;`

Would be a way to destructure this with default values.

Reply:
Done.
Thank you!